repo/packages/C/confluent-kafka/10/command.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Gotta love "Chrome" ... it seemed to be directing me to the later comment :(

Our quick poll of the customer base showed folks still using DC/OS 1.7.   We'll set the expectation that this will be the last Confluent package version that supports DC/OS 1.7 and make the change to DC/OS 1.8 and later with our 2017 releases.